rootx/src/rootxx.cxx

```diff
@@ -779,8 +780,9 @@ int DrawCredits(bool draw, bool extended)
 
       struct passwd *pwd = getpwuid(getuid());
       if (pwd) {
-         char *name = new char [strlen(pwd->pw_gecos)+1];
-         strcpy(name, pwd->pw_gecos);
+         size_t sz = strlen(pwd->pw_gecos)+1;
+         char *name = new char [sz];
+         strlcpy(name, pwd->pw_gecos, sz);
```

Comment:
Funnily, I find no practical difference between the old and new code ... because `sz` is anyway th result of strlen ...

Humm ... I guess it would make a difference if the input can be changed between the strlen and the strcpy ...